pkg/interfaces/contracts/vault/IAggregatorCompositeLiquidityRouter.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { ICompositeLiquidityRouterQueries } from "./ICompositeLiquidityRouterQueries.sol";
+
+/**
+ * @notice The aggregator composite liquidity router supports add/remove liquidity operations on ERC4626 and nested pools.
+ * @dev This contract allow interacting with ERC4626 Pools (which contain wrapped ERC4626 tokens) using only underlying
+ * standard tokens. For instance, with `addLiquidityUnbalancedToERC4626Pool` it is possible to add liquidity to an
+ * ERC4626 Pool with [waDAI, waUSDC], using only DAI, only USDC, or an arbitrary amount of both. If the ERC4626 buffers
+ * in the Vault have liquidity, these will be used to avoid wrapping/unwrapping through the wrapped token interface,
+ * saving gas.
+ *
+ * For instance, adding only DAI to the pool above (and assuming an aDAI buffer with enough liquidity), would pull in
+ * the DAI from the user, swap it for waDAI in the internal Vault buffer, and deposit the waDAI into the ERC4626 pool:
+ * 1) without having to do any expensive ERC4626 wrapping operations; and
+ * 2) without requiring the user to construct a batch operation containing the buffer swap.
+ *
+ * The aggregator composite liquidity router uses prepayment instead of permit2.
```

Comment:
```suggestion
 * The aggregator composite liquidity router is designed to be called from a contract vs. an EOA through a UI.
 * It uses prepayment instead of permit2, and defines identical functions with the same parameters as
 * `ICompositeLiquidityRouter`, but without support for native ETH wrapping/unwrapping.
```